modules/Bio/EnsEMBL/Analysis/Hive/Config/GenblastHomology.pm

```diff
@@ -0,0 +1,532 @@
+
+    ensembl_analysis_script           => catdir($self->o('enscode_root_dir'), 'ensembl-analysis', 'scripts'),
+    remove_small_orf_script             => catfile($self->o('ensembl_analysis_script'), 'genebuild', 'remove_small_orf.pl'),
+  
+    # cutoffs for removing small_orf genes
+    'small_orf_cutoff' => '100',
+    'intron_cutoff' => '75',
+
+########################
+# Extra db settings
+########################
+
+    'num_tokens' => 10,
+    mysql_dump_options => '--max_allowed_packet=1000MB',
+
+########################
+# Executable paths
+########################
+
+    'blast_type' => 'ncbi', # It can be 'ncbi', 'wu', or 'legacy_ncbi'
+    'exonerate_path'         => catfile($self->o('software_base_path'), 'opt', 'exonerate09', 'bin', 'exonerate'),
+    'uniprot_genblast_batch_size' => 15,
+    'uniprot_table_name'          => 'uniprot_sequences',
+    genewise_path => catfile($self->o('binary_base'), 'genewise'),
+    'genblast_path'     => catfile($self->o('binary_base'), 'genblast'),
+    'genblast_eval'     => $self->o('blast_type') eq 'wu' ? '1e-20' : '1e-1',
+    'genblast_cov'      => '0.5',
+    'genblast_pid'      => '30',
+    'genblast_max_rank' => '5',
+    'genblast_flag_small_introns' => 1,
+    'genblast_flag_subpar_models' => 0,
+
+
+##################################
+# Memory settings for the analyses
+##################################
+    'default_mem'          => '900',
+    'genblast_mem'         => '1900',
+    'genblast_retry_mem'   => '4900',
+
+    
+########################
+# db info
+########################
+
+    'genblast_db' => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_genblast_'.$self->o('release_number'),
+      -host   => $self->o('genblast_db_server'),
+      -port   => $self->o('genblast_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+
+    'genblast_nr_db' => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_genblast_nr_'.$self->o('release_number'),
+      -host   => $self->o('genblast_db_server'),
+      -port   => $self->o('genblast_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'killlist_db' => {
+      -dbname => $self->o('killlist_db_name'),
+      -host   => $self->o('killlist_db_server'),
+      -port   => $self->o('killlist_db_port'),
+      -user   => $self->o('user_r'),
+      -pass   => $self->o('password_r'),
+      -driver => $self->o('hive_driver'),
+    },
+
+  };
+}
+
+sub pipeline_create_commands {
+    my ($self) = @_;
+
+    return [
+    # inheriting database and hive tables' creation
+      @{$self->SUPER::pipeline_create_commands},
+
+      $self->hive_data_table('protein', $self->o('uniprot_table_name')),
+      
+      'mkdir -p '.$self->o('genome_dumps'),
```

Comment:
This can be removed. The directory will be created by a different pipeline and we would only read from the file already created

Reply:
Changes effected